### PR TITLE
feat: rel: add signed ops and its test

### PR DESCRIPTION
**Pull request by Akas1337**

### CODE REVIEW

File: src/circuits/etable_compact/op_configure/op_rel.rs [hunk 1]
```diff
@@ -35,7 +35,9 @@ pub struct RelConfig {
     _lhs_leading_bit: BitCell,
     _rhs_leading_bit: BitCell,
     _lhs_rem_value: CommonRangeCell,
+    _lhs_rem_diff: CommonRangeCell,
     _rhs_rem_value: CommonRangeCell,
+    _rhs_rem_diff: CommonRangeCell,
```
Comment (junyu0312) [inline src/circuits/etable_compact/op_configure/op_rel.rs:40]: remove underline

File: src/circuits/etable_compact/op_configure/op_rel.rs [hunk 2]
```diff
@@ -311,6 +352,19 @@ impl<F: FieldExt> EventTableOpcodeConfig<F> for RelConfig {
             self.is_eight_bytes.assign(ctx, true)?;
         }
 
+        if vec![RelOp::SignedGt, RelOp::SignedGe, RelOp::SignedLt, RelOp::SignedLe].contains(&class) {
+            let shift: usize = if vtype == VarType::I64 {I64_REM_SHIFT} else {I32_REM_SHIFT};
+            self.op_is_sign.assign(ctx, true)?;
+            let left: u64 = lhs >> shift;
```
Comment (junyu0312) [inline src/circuits/etable_compact/op_configure/op_rel.rs:358]: let left_leading_u4 =

Comment (Akas1337) [reply src/circuits/etable_compact/op_configure/op_rel.rs]: fixed

File: src/circuits/etable_compact/op_configure/op_rel.rs (hunk 1 above)
Comment (Akas1337) [reply src/circuits/etable_compact/op_configure/op_rel.rs]: fixed

**Review by junyu0312 — APPROVED** [review]

**Pull request merged**